src/library/Api/API.js

```diff
@@ -167,25 +221,25 @@ const API = {
             },
             body: body,
         })
-        .then((response) => {
-            return response.json();
-          })
-        .then((response) => {
-            // If the response is an error, call the error handler
-            if (response.error) {
-                if (typeof errorHandler === 'function') {
-                    errorHandler(response.error);
-                } else {
-                    console.error(`${response.error.message} (Code: ${response.error.code})`);
-                    console.warn("No error handler was specified. The error was logged to the console. Documentation: https://fossbilling.org/docs/api/javascript   ");
+            .then((response) => {
+                return response.json();
+            })
+            .then((response) => {
+                // If the response is an error, call the error handler
+                if (response.error) {
+                    if (typeof errorHandler === 'function') {
+                        errorHandler(response.error);
+                    } else {
+                        console.error(`${response.error.message} (Code: ${response.error.code})`);
+                        console.warn("No error handler was specified. The error was logged to the console. Documentation: https://fossbilling.org/docs/api/javascript   ");
```

Comment:
While you’re at it, can you remove the blank space here?

Reply:
Changed it